include/Core/Controller.php

```diff
@@ -131,11 +132,11 @@ public function verify_get_request_nonce( $result, $action ) {
 	}
 
 	/**
-	 * Ajax call back function for getting taxonomies to load select2 options.
+	 * Ajax call back function to load taxonomy terms by taxonomy.
 	 *
 	 * @since 6.0.0
 	 */
-	public function load_taxonomy_term() {
+	public function load_taxonomy_terms() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [load_taxonomy_terms accesses the super-global variable $_GET.](https://app.codacy.com/app/BulkWP/bulk-delete/pullRequest?prid=1939930)